.schema/config.schema.json

```diff
@@ -1459,6 +1504,38 @@
             }
           ]
         },
+        "forward": {
```

Comment:
I would prefer calling this `remote_json` similar to the authorizer, as we expect a JSON response from the upstream! :)

Reply:
Renamed the authenticator into `remote_json`